### DISCUSSION TITLE
SincInterpolationDft.m in Q81493 is missing!

### DISCUSSION THREAD

**Discussion by SuperStarCZ** [RoyiAvital/StackExchangeCodes#10] [Show and tell]
Can you provide the file "SincInterpolationDft.m" in Q81493?

thx!

**Comment by RoyiAvital** [comment]
Do you mean for https://dsp.stackexchange.com/questions/81493 ?

I removed it as it seems no one is interested in it (No up votes or any interest).

**Comment by SuperStarCZ** [comment]
Yes, I ran the codes you provided. However, I found this file was missing. Can you provide this file in the solution?
